### ISSUE TITLE
$digest error

### ISSUE THREAD

**Issue by vinsonyung** [theaqua/ng-selectize#2]
Hi author & others,

I got "Error: [$rootScope:inprog] http://errors.angularjs.org/1.4.0-rc.2/$rootScope/inprog?p0=%24digest" after selected an option.

HTML:
```
<select selectize="selectizeProducts" options="products" ng-model="product_id"></select>
```

AngularJs:
```
$scope.products = [];

$http.get('/product/get_list').success(function(data, status, headers, config) {
    $scope.products = data;
});
$scope.selectizeProducts = {
		valueField: 'id',
		labelField: 'name',
		searchField: 'name',
		placeholder: "Please select...",
		onChange: function(id) {
			getProductInfo(id);
		}
	};
```

**Comment by Anahkiasen** [comment]
Same here

**Comment by theaqua** [comment]
Sorry, guys, at now I don't have time for digging that problem, can anyone make PR, if have fix?

**Comment by nixolas1** [comment]
Remember to use <input> elements for multiselects, not <select>.
That fixes it for multiselect.

**Comment by yakovkhalinsky** [comment]
I've had to resolve this for our team.

PR coming soon :+1:

**Comment by yakovkhalinsky** [comment]
For the people following this issue #3 has been merged and version 0.0.4 from bower will include this $digest fix :+1:

**Comment by theaqua** [comment]
Yep. Thanks, @yakovkhalinsky.
